distribution/docker/solrcloud/pom.xml

```diff
@@ -0,0 +1,132 @@
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.distribution</groupId>
+        <artifactId>docker</artifactId>
+        <version>2.26.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>ddf-solrcloud</artifactId>
+    <groupId>ddf.distribution.docker</groupId>
+    <packaging>pom</packaging>
+    <name>DDF :: Distribution :: Docker :: SolrCloud</name>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources-filtered</directory>
+                <filtering>true</filtering>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <!-- avoid directory clean failures on Windows due to exec command holding onto directory lock -->
+                    <failOnError>false</failOnError>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>up</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>docker-compose</executable>
+                            <arguments>
+                                <argument>up</argument>
+                                <argument>-d</argument>
+                            </arguments>
+                            <workingDirectory>${project.build.outputDirectory}</workingDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>logs</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>docker-compose</executable>
+                            <arguments>
+                                <argument>logs</argument>
+                                <argument>--tail=50</argument>
```

Comment:
❔ Just curious, How did you come up with 50 line for tailing the logs?

Reply:
I used to use 100 but that tends to be way more context than I ever needed when tailing the logs initially.  So I reduced it to 50 so it did not spam the terminal as much.  Nothing scientific.  Just want to provide enough context in case the tail of the logs starts in the middle of stack trace for example.